db/QuestionConfirmation.js

```diff
@@ -0,0 +1,47 @@
+const db = require("./");
+
+const QuestionConfirmation = () => db("QuestionConfirmations");
+
+const findById = id =>
```

Comment:
Probably best to follow pattern of other exports and change this to `module.exports.findById = id => ....`

Reply:
As discussed changed to one use of `module.exports`